applications/luci-app-unbound/luasrc/model/cbi/unbound.lua

```diff
@@ -0,0 +1,126 @@
+-- Copyright 2008 Steven Barth <steven@midlink.org>
+-- Copyright 2016 Eric Luehrsen <ericluehrsen@hotmail.com>
+-- Licensed to the public under the Apache License 2.0.
+
+m = Map("unbound", translate("Recursive DNS"),
+	translate("Unbound is a validating, recursive, and caching DNS resolver."))
+	
+s = m:section(TypedSection, "unbound", translate("Unbound Settings"))
+s.addremove = false
+s.anonymous = true
+
+s:tab("service", translate("Unbound Service"))
+s:tab("resource", translate("Unbound Resources"))
+s:tab("dnsmasq", translate("Dnsmasq Link"))
+
+--Enable Unbound
+
+e = s:taboption("basic", Flag, "enabled", translate("Enable Unbound:"),
+  translate("Enable the initialization scripts for Unbound."))
+e.rmempty = false
+
+function e.cfgvalue(self, section)
+	return luci.sys.init.enabled("unbound") and self.enabled or self.disabled
+end
+
+function e.write(self, section, value)
+	if value == "1" then
+		luci.sys.init.enable("unbound")
+		luci.sys.call("/etc/init.d/unbound start >/dev/null")
+	else
+		luci.sys.call("/etc/init.d/unbound stop >/dev/null")
+		luci.sys.init.disable("unbound")
+	end
+
+	return Flag.write(self, section, value)
+end
+
+--Service Tab
+
+mcf = s:taboption("service", Flag, "manual_conf", translate("Manual Conf:"),
+  translate("Skip UCI and use the Unbound raw configuration in /etc/unbound/unbound.conf."))
+mcf.rmempty = false
+
+eds = s:taboption("service", Value, "edns_size", translate("EDNS Size:"),
+  translate("Extended DNS packet size limit to manange MTU restrictions."))
+eds.datatype = "and(uinteger,min(512),max(4096))"
+eds.rmempty = false
+
+prt = s:taboption("service", Value, "listen_port", translate("Listening Port:"),
+  translate("Unbound will listen for queries on this port."))
+prt.datatype = "and(uinteger,53,min(1024),max(8192))"
```

Comment:
Yes sorry ... tested, failed, and the page overall was ugly. I fouled up rotating from my test branch to this pull branch. I actually decided to just let it be "port."